### PR TITLE
admin_namespace

### PR DESCRIPTION
**Pull request by gdhaison**

### CODE REVIEW

File: app/controllers/admin_controller.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  before_action :verify_admin!
+  
+  def index; end
+
+  private
+
+  def verify_admin!
+    return if current_user.admin?
+    flash[:danger] = I18n.t "access_denied"
+  end
```
Comment (N-Viet-ruby-dev) [inline app/controllers/admin_controller.rb:13]: Thêm redirect_to login page

Comment (gdhaison) [reply app/controllers/admin_controller.rb:13]: oke anh !!

File: app/views/layouts/_header.html.erb [hunk 2]
```diff
@@ -1,10 +1,19 @@
...
-    <li><%= link_to t(".settings"),edit_user_path(current_user) %></li>
-    <%= link_to t(".logout"), logout_path, method: :delete %>
-  <% else %>
-    <li><%= link_to t(".login"),login_path %></li>
-  <% end %>
-</ul>
+<nav class="navbar navbar-inverse">
+  <div class="container-fluid">
+    <div class="navbar-header">
+      <a class="navbar-brand" href="#">WebSiteName</a>
```
Comment (duongpham910) [inline app/views/layouts/_header.html.erb]: i18n

File: app/views/layouts/_header.html.erb [hunk 3]
```diff
@@ -1,10 +1,19 @@
...
+    </div>
+    <ul class="nav navbar-nav">
+      <li class="active"><a href="#">Home</a></li>
+      <% if logged_in?%>
```
Comment (duongpham910) [inline app/views/layouts/_header.html.erb]: convention

File: app/views/layouts/_header.html.erb [hunk 4]
```diff
@@ -1,10 +1,19 @@
...
+        <li><%= link_to t(".user"),users_path %></li>
+        <li><%= link_to t(".profile"),current_user %></li>
+        <li><%= link_to t(".settings"),edit_user_path(current_user) %></li>
+        <li><%= link_to t(".logout"), logout_path, method: :delete %></li>
+      <% else %>
+      <li><%= link_to t(".login"),login_path %></li>
```
Comment (duongpham910) [inline app/views/layouts/_header.html.erb]: indent

**Comment by duongpham910** [comment]
LGTM

**Pull request merged**